addons/barcodes/__manifest__.py

```diff
@@ -17,6 +17,7 @@
         ],
         'web.qunit_suite_tests': [
             'barcodes/static/tests/barcode_tests.js',
+            'barcodes/static/tests/float_scannable_field_test.js',
```

Comment:
maybe check if we can instead defines assets with a *:
```
barcodes/static/tests/**/*
```
so we don't have to do that each time